cmd/server-main.go

```diff
@@ -209,10 +209,10 @@ func initServerConfig(c *cli.Context) {
 	minioInit(c)
 
 	// Create certs path.
-	fatalIf(createCertsPath(), "Unable to create \"certs\" directory.")
+	fatalIf(createConfigDir(), "Unable to create \"certs\" directory.")
```

Comment:
I think it is better to move createConfigDir() inside minioInit() since config dir is already created there, it would make sense to put all creation process in one place